charts/artifact-hub/values.schema.json

```diff
@@ -42,6 +42,11 @@
                     "default": "5432",
                     "type": "string"
                 },
+                "sslmode": {
```

Comment:
I added most of the new properties I introduced to the corresponding `required` sections. However, the logic behind many of the properties currently marked as `required` seemed a bit inconsistent to me. For example:

- `hub.ingress.enabled` is a required property. If the user omits this property, it would still fallback to its default. And even if there was no default value specified, the expression `{{- if .Values.hub.ingress.enabled -}}` would still evaluate to `false`. So this IMHO no required value. (The same logic applies to many other properties like `db.user`,  `db.port`, etc.)
- `hub.server.oauth` is not a required property. However, expressions like `{{- if .Values.hub.server.oauth.github.enabled }}` would probably break with some kind of `Nil` reference error.

Reply:
Thanks!

The problem with the default values is that they may not always be applied during an upgrade (i.e. when using the `reuse-values` flag). But you make a good point about that boolean expression evaluating to `false` when the value is missing.

Regarding the `hub.server.oauth` case, this was a mistake and not intentional, good catch!